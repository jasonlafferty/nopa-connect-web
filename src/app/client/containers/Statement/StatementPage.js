import React from 'react';
import { Layout } from '../../components';

const StatementPage = (props) => {
  return (
    <Layout></Layout>
  );
}

export default StatementPage;
