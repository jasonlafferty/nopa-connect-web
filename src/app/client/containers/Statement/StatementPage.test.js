import React from 'react';
import StatementPage from './StatementPage';
import renderer from 'react-test-renderer';

test('Renders correctly', () => {
  const tree = renderer.create(
    <StatementPage />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
