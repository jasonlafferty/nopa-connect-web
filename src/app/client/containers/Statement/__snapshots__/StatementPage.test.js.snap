// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div>
  <div />
  <div
    className="main-content"
  />
  <div />
</div>
`;
